src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -180,18 +180,28 @@
  * to be designed for extension.
  * </p>
  * <pre>
- * &lt;module name=&quot;DesignForExtension&quot;&gt;
+ * &lt;module name=&quot; DesignForExtension &quot;&gt;
  *   &lt;property name=&quot;ignoredAnnotations&quot; value=&quot;Override, Test&quot;/&gt;
- * &lt;/module&gt;
+ * &lt;/module&gt; // Checks for ignoredAnnotations
  * </pre>
  * <pre>
  * public class A {
  *   &#64;Override
  *   public int foo() {
- *     return 2;
+ *     return 2;         // OK
  *   }
  *
- *   public int foo2() {return 8;} // violation
+ *   public int foo2() {return 8;}  // violation, required annotation is not present .
+ *
+ *
+ *   &#64;Override
+ *   public int foo3() {
+ *       return 9 ;         // OK
+ *    }
+ *
+ *    public int foo() {return 7;}  // violation, required annotation is not present .
```

Comment:
please make changes to examples to make them same for java code in all examples.
the only diffrence should be is trailing comments that changing due to different config.

example: https://checkstyle.org/config_annotation.html#AnnotationLocation_Examples